suspendmanager.lua

```diff
@@ -531,3 +557,42 @@ end
 if not dfhack_flags.module then
     main({...})
 end
+
+-- Overlay Widget
+JobOverlay = defclass(JobOverlay, overlay.OverlayWidget)
+JobOverlay.ATTRS{
+    default_pos={x=-41,y=14},
+    default_enabled=true,
+    viewscreens='dwarfmode/ViewSheets/BUILDING',
+    frame={w=30, h=5},
+    frame_style=gui.MEDIUM_FRAME,
+    frame_background=gui.CLEAR_PEN,
+}
+
+function JobOverlay:init()
+    self:addviews{
+        widgets.Label{
+            frame={t=0, l=0},
+            text='Staying suspended:'
+        },
+        widgets.Label{
+            frame={t=2, l=0},
+            text={'', {text=self:callback('get_reason_string')}}
+        }
+    }
+end
+
+function JobOverlay:get_reason_string()
+    return Instance:suspensionDescription(dfhack.gui.getSelectedJob())
+end
+
+function JobOverlay:render(dc)
+    if not isEnabled() or not self:get_reason_string() then
+        return
+    end
+    JobOverlay.super.render(self, dc)
+end
+
+OVERLAY_WIDGETS = {
+    inspector=JobOverlay
```

Comment:
maybe "status" instead of "inspector" (actually, "status" would have been a better name for the `buildingplan` overlay too..)

Reply:
Can you spot the mindless copy/paste? :D